logging/images/logstash/logstash.conf

```diff
@@ -0,0 +1,13 @@
+input {
+  syslog { port => 51415 }
+}
+
+output {
+  elasticsearch {
+    hosts => ["elasticsearch:9200"]
+  }
+  # Remove in production
```

Comment:
gotcha - I saw another example that was more complex in one of your repos that I have yet to try to understand - would that be a better fit?

https://github.com/vfarcic/cloud-provisioning/blob/master/conf/logstash-proxy.conf